NEWS.md

```diff
@@ -1,89 +1,109 @@
-# ggplot2 3.0.0.9000
+# ggplot2 3.1.0
 
-*   `geom_boxplot()` now understands the `width` parameter even when used with
-    a non-standard stat, such as `stat_identity()` (@clauswilke, #2893).
-
-*   `scale_*_date()`, `scale_*_time()` and `scale_*_datetime()` can now display 
-    a secondary axis that is a __one-to-one__ transformation of the primary axis,
-    implemented using the `sec.axis` argument to the scale constructor 
-    (@dpseidel, #2244).
+## Breaking changes
 
-*   The error message in `compute_aesthetics()` now provides the names of only
-    aesthetics with mismatched lengths, rather than all aesthetics (@karawoo,
-    #2853).
-    
-*   `coord_sf()` now respects manual setting of axis tick labels (@clauswilke,
-    #2857, #2881).
-    
-*   `coord_sf()` now accepts two new parameters, `label_graticule` and `label_axes`,
-    that can be used to specify which graticules to label on which side of the plot
-    (@clauswilke, #2846).
+This is a minor release and breaking changes have been kept to a minimum. End users of ggplot2 are unlikely to encounter any issues. However, there are a few items that developers of ggplot2 extensions should be aware of. For additional details, see also the discussion accompanying issue #2890.
+
+*   In non-user-facing internal code (specifically in the `aes()` function and in
```

Comment:
Switch "of" to "for" for "British spelling for aesthetics…"? (I had a split second of wondering how the British spell the word of aesthetics)

Reply:
Ok.